tests/unit/type/string.tcl

```diff
@@ -461,12 +461,14 @@ start_server {tags {"string"}} {
     }
     
     test {trim on SET with big value} {
+    if {[string match {*jemalloc*} [s mem_allocator]]} {
```

Comment:
Let's reorder the lines so that the entire test is skipped (won't look as if it passed)

Reply:
Done.

> I'm confused. This fix isn't for the same test that was mentioned to fail in [#11817 (comment)](https://github.com/redis/redis/pull/11817#discussion_r1124569311)

I forgot about the module-related tests, they all fail for the same reason.